pages/linux/init.md

```diff
@@ -0,0 +1,23 @@
+# init
+
+> Init is a daemon process that start during booting continue to run until the system is shut down.
```

Comment:
No need to mention "Init" in the sentence. It is expected that the main heading will be about the tool itself. I recommend a slight change - 
```
A daemon, which is the first process to be started by the kernel, and continues to run until the system is shut down.
```